### PR TITLE
Update istio.io/client-go to 1.9.4

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by shamsher31 — COMMENTED** [review]
Can we use https://github.com/istio/client-go/releases/tag/1.9.5 as we are already using `1.9.5` as the base Istio version?

**Comment by luksa** [comment]
We can't, because the tag is wrong. Should be v1.9.5, not 1.9.5.

**Review by shamsher31 — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -6,14 +6,12 @@ require (
...
-	google.golang.org/protobuf v1.25.0 // indirect
-	istio.io/client-go v1.8.1
-	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.19.2
-	k8s.io/client-go v0.19.2
+	istio.io/client-go v1.9.4
+	k8s.io/api v0.20.1
+	k8s.io/apimachinery v0.20.1
+	k8s.io/client-go v0.20.1
 	k8s.io/code-generator v0.19.2
```
Comment (rcernich) [inline go.mod:13]: Did you want to update this one too?

**Review by jwendell — APPROVED** [review]

**Comment by rcernich** [comment]
Unfortunately, it looks like the maistra/istio 1.9 branch is using 1.9.5, which had changes to MeshConfig api.

**Comment by luksa** [comment]
OK, then we have to refer to the commit instead of the tag. See https://github.com/maistra/xns-informer/pull/16

**Pull request merged**